zebra/main.c

```diff
@@ -65,6 +65,8 @@ struct mgmt_be_client *mgmt_be_client;
 /* Route retain mode flag. */
 int retain_mode = 0;
 
+bool fpm_pic_nexthop = true;
```

Comment:
What is this variable used for? It is set to true here and never changed. It seems that it is unnecessary and can be removed?

Reply:
fixed